src/ops/ops.go

```diff
@@ -195,6 +196,27 @@ func (o *ops) Reboot() error {
 	return nil
 }
 
+func (o *ops) SetBootOrder(device string) error {
+	_, err := o.ExecPrivilegeCommand(o.logWriter, "test", "-d", "/sys/firmware/efi")
+	if err != nil {
+		o.log.Info("efi not supported")
+		return nil
+	}
+
+	o.log.Info("Setting efibootmgr to boot from disk")
+	_, err = o.ExecPrivilegeCommand(o.logWriter, "mount", device+"2", "/mnt")
+	if err != nil {
+		o.log.Errorf("Failed to mount device %s, err: %s", device+"2", err)
+		return err
+	}
+	_, err = o.ExecPrivilegeCommand(o.logWriter, "efibootmgr", "-d", device, "-p", "2", "-c", "-L", "RHCOS", "-l", "\\EFI\\BOOT\\BOOTX64.EFI")
```

Comment:
Does coreos-installer always install on the second partition of a device? Please add a comment explaining it.

Reply:
Added a comment!